### PR TITLE
Add "Contribute" section to sidebar

### PR DESCRIPTION
**Pull request by rnevius**
## Description

As above

## Motivation and Context

Updates #2443:

> Can we add a global widget somewhere near the top on every page that would include our invitations to contribute (e.g. "Edit this page" and "Open a docs issue")? Again https://grafana.com/docs/grafana/latest/installation/requirements/ has a good example just above the "On this page:" menu.

### CODE REVIEW

File: layouts/partials/contribute.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+          Edit this page
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/sensu/sensu-docs/issues/new/choose" target="_blank">
+          <i class="fa fa-bug" aria-hidden="true"></i>
+          Report a docs issue
+        </a>
+      </li>
+    </ul>
```
Comment (hillaryfraley) [inline layouts/partials/contribute.html:23]:
```suggestion
      <li>
        <a href="https://github.com/sensu/sensu-docs/blob/master/CONTRIBUTING.md" target="_blank">
          <i class="fa fa-info-circle" aria-hidden="true"></i>
          Read the contributing guide
        </a>
      </li>
    </ul>
```

Comment (hillaryfraley) [reply layouts/partials/contribute.html:23]: Just pulling the contributing guide and GH star button up into the new sidebar item too.

**Review by hillaryfraley — COMMENTED** [review]
@rnevius this looks good. I think we would want to remove lines 14 though 17 from layouts/partials/drawer.html too -- wdyt?

```
        <iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-docs&type=star&count=true" frameborder="0" scrolling="0" width="170" height="20"></iframe>
        <a target="_blank" href="https://github.com/sensu/sensu-docs/issues/new?title=Docs%20Feedback&body=%3C!--Thanks%20for%20submitting%20feedback%20to%20the%20Sensu%20docs!--%3E%0A%0ALink%20to%20page%3A%20%0A%0A%23%23%23%20Feedback">Open a docs issue</a>
        <a target="_blank" href="{{.Site.Params.repo_url}}/edit/master/content/{{replace .File.Path "latest" .Site.Params.products.sensu_go.latest}}">Edit this page</a>
        <a target="_blank" href="https://github.com/sensu/sensu-docs/blob/master/CONTRIBUTING.md">Read the contributing guide</a>
```

**Comment by rnevius** [comment]
@hillaryfraley fine by me, but the sidebar content isn't displayed on mobile (while the drawer content is). If this content shouldn't be shown at all on mobile, it can be safely removed from the drawer template.

**Comment by hillaryfraley** [comment]
@rnevius Yes, that's a good reason to leave the drawer links there. @cwjohnston do you agree with keeping the drawer links (they are listed at the bottom left, after the left-nav) along with this new sidebar (which will include the same links)?

I am thinking maybe we only include the GH star button in the drawer if we keep both. Keeps the sidebar a tiny bit cleaner. I'm still looking at both and trying to decide...

**Comment by hillaryfraley** [comment]
OK -- I think I will leave the GH button out in the new sidebar. I'll get this reviewed now.

**Review by hillaryfraley — APPROVED** [review]
I added the contributing guide link -- this looks good to me!

**Comment by hillaryfraley** [comment]
Merged! And @rnevius I believe this is the last part of https://github.com/sensu/sensu-docs/issues/2443 🎉 
Thank you for all of your work on this.

**Pull request merged**